ide-eclipse.html

```diff
@@ -0,0 +1,366 @@
+<h2>JavaFX <span class="JFX">11</span> and Eclipse</h2>
+
+<p>This section explains how to use Java <span class="JDK">11</span> and JavaFX <span class="JFX">11</span> from Eclipse. 
+    Version Eclipse 2018-09 (4.9) Build id: I20180906-0745 was used for the following screenshots.
+</p>
+
+<p>
+    Download an appropriate <a href="https://jdk.java.net/11/">JDK <span class="JDK">11</span></a> for your operating system. 
```

Comment:
A better approach would have been to write JDK 11 and point them to the latest JDK 11 to download.

Reply:
Same as above, if we use `JDK_MAJOR`, that will be 11, and the link will always show the latest version available.